packages/wc-registry/package.json

```diff
@@ -18,7 +18,7 @@
     "build": "npm run build:graphql && npm run build:ts && npm run build:copy",
     "build:ts": "../../node_modules/.bin/tsc",
     "build:copy": "cp -r out/{src,gen}/* .",
-    "build:graphql": "graphql-codegen",
+    "build:graphql": "graphql-codegen -c codegen.yml",
```

Comment:
weird, I didn't need this.

Reply:
🤷‍♂️ yeah i thought it was odd, too. 

We'll probably want to run it from the root with a `.graphqlrc` anyways at some point